deployment/deployment.go

```diff
@@ -183,9 +189,141 @@ func Update(id, cloudRole string, args map[string]string, client *houston.Client
 
 	d := r.Data.UpdateDeployment
 	tab := newTableOut()
-	tab.AddRow([]string{d.Label, d.ReleaseName, d.Version, d.Id}, false)
+	tab.AddRow([]string{d.Label, d.ReleaseName, d.Version, d.Id, d.AirflowVersion}, false)
 	tab.SuccessMsg = "\n Successfully updated deployment"
 	tab.Print(out)
 
 	return nil
 }
+
+// Upgrade airflow deployment
+func AirflowUpgrade(id, desiredAirflowVersion string, client *houston.Client, out io.Writer) error {
+	if desiredAirflowVersion == "" {
+		selectedVersion, err := getAirflowVersionSelection(id, client, out)
+		if err != nil {
+			return err
+		}
+		desiredAirflowVersion = selectedVersion
+	}
+
+	vars := map[string]interface{}{"deploymentId": id, "desiredAirflowVersion": desiredAirflowVersion}
+
+	req := houston.Request{
+		Query:     houston.UpdateDeploymentAirflowRequest,
+		Variables: vars,
+	}
+
+	r, err := req.DoWithClient(client)
+	if err != nil {
+		return err
+	}
+
+	d := r.Data.UpdateDeploymentAirflow
+	tab := &printutil.Table{
+		Padding:        []int{30, 30, 10, 50, 10},
+		DynamicPadding: true,
+		Header:         []string{"NAME", "DEPLOYMENT NAME", "ASTRO", "DEPLOYMENT ID", "AIRFLOW VERSION"},
+	}
+	tab.AddRow([]string{d.Label, d.ReleaseName, "v" + d.Version, d.Id, d.AirflowVersion}, false)
+
+	tab.SuccessMsg = fmt.Sprintf("\nThe upgrade from Airflow %s to %s has been started.", d.AirflowVersion, d.DesiredAirflowVersion) +
+		"To complete this process, replace the image referenced in your Dockerfile and deploy to Astronomer.\n" +
```

Comment:
@andriisoldatenko Whoops, messaged this above but how about:

```suggestion
		"To complete this process, add an Airflow %s image to your Dockerfile and deploy to Astronomer.\n" +
```

`Airflow %s` = "Desired Airflow Version"